notifications_utils/jinja_templates/email_template.jinja2

```diff
@@ -55,7 +55,7 @@
                <img
                 src="https://assets.notification.canada.ca/gc-logo-fr.png?20210531"
                 alt="Gouvernement du Canada / Government of Canada"
-                height="70"
+                height="55"
```

Comment:
Same height than the GoC EN. https://assets.notification.canada.ca/gc-logo-fr.png has the appropriate dimensions